parent/pom.xml

```diff
@@ -147,7 +147,7 @@
         <yammer.metrics.core.version>2.2.0</yammer.metrics.core.version>
         <yardstick.version>0.8.3</yardstick.version>
         <zkclient.version>0.5</zkclient.version>
-        <zookeeper.version>3.6.3</zookeeper.version>
+        <zookeeper.version>3.9.1</zookeeper.version>
```

Comment:
I would suggest upgrading this dependency to the 3.8.3 version. In accordance with https://zookeeper.apache.org/releases.html, this is the last stable version.

Reply:
Hi @nao-it ,

Yep, it seems that I was wrong about compilation. Thank you for your efforts!